prboom2/src/SDL/i_main.c

```diff
@@ -121,6 +128,11 @@ static void I_SignalHandler(int s)
           buf, signal_context);
 }
 
+static void I_IntHandler(int s)
```

Comment:
Could you tell me programmatically what happens when this function is called and when it returns? Does the program execution jump here and then back to the instruction it was on when it got the signal?

Reply:
Yes, essentially.  The OS arranges for the signal handler to run in a new frame on the call stack, after which execution resumes at precisely the point where the signal was delivered.